ansible/roles/distributed_press/templates/distributed.press.service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Distributed Press API Server
+After=network-online.target
+[Service]
+WorkingDirectory={{distributed_press_source}}/v1
+User={{distributed_press_user}}
+ExecStart=npx ts-node-esm index.ts --port {{distributed_press_port}} --host {{distributed_press_host}} --ipfs-provider {{distributed_press_ipfs_provider}}
+Restart=always
+RestartSec=3
+LimitNOFILE=4096
```

Comment:
Do we need to bump this? (see: https://www.freedesktop.org/software/systemd/man/systemd.exec.html#Process%20Properties) Seems like potentially unsafe to bump beyond 1024

Reply:
Not sure actually. @hyphacoop/infrastructure any thoughts?

Reply:
I think the key in the comment I think your referring to is 
> Typically applications should increase their soft limit to the hard limit on their own
Which I would assume the app does not do.

 I don't see the danger other then not using `select(2)` and possible resource utilization (DDOS). In un-managed code I can see leaks of not closing descriptors when your done with them, but with a GC I don't think its much of a danger if leakage.

I have seen this value set much higher in other apps esp distributed systems I have seen this value set much higher then 4096. For example in Saturn the docker container is set to 10,000 (but i guess that is system wide not for a single process)
https://github.com/filecoin-saturn/L1-node/blob/main/docker-compose.yml#L12-L15

Keep in mind this is the `Distributed Press API Server` its going to be touching lots of file descriptors, form incoming sockets, to outgoing connections to reading files and metadata.

Just my 2 cents.

Reply:
K, let's keep it as is then and revisit it if it's a problem. :+1: